railties/lib/rails/generators/rails/scaffold_controller/templates/controller.rb.tt

```diff
@@ -53,7 +53,8 @@ class <%= controller_class_name %>Controller < ApplicationController
   private
     # Use callbacks to share common setup or constraints between actions.
     def set_<%= singular_table_name %>
-      @<%= singular_table_name %> = <%= orm_class.find(class_name, "params[:id]") %>
```

Comment:
`find` already raise `ActiveRecord::NotFoundError`. Why are you changing to `find_by_id` and raising it by hand?

Reply:
diegopiccinini told me, sorry, forget it.